src/content/en/android/trusted-web-activity/whats-new/index.md

```diff
@@ -0,0 +1,461 @@
+project_path: /web/android/_project.yaml
+book_path: /web/android/_book.yaml
+description: Learn about the new features for web applications that use Trusted Web Activity.
+
+{# wf_published_on: 2020-11-25 #}
+{# wf_updated_on: 2020-11-25 #}
```

Comment:
```suggestion
{# wf_published_on: 2020-12-02 #}
{# wf_updated_on: 2020-12-02 #}
```

Reply:
If we don't set the date a little ahead, this article won't be first in the list when it comes out.